src/CMakeLists.txt

```diff
@@ -493,6 +501,14 @@ function(add_doom_tiny SUFFIX RENDER_LIB)
             -Wno-format-truncation)
     target_compile_definitions(doom_tiny${SUFFIX} PRIVATE
         DOOM_TINY=1
+
+        # LILYGO_TTGO=1
+        # ST7789_240_135=1
+        # SSD1306_70_40=1
+        # SSD1306_70_40_i2c=1
+        ${SCREEN}=1
+        
+        SUPPORT_OVERLAYS=1
```

Comment:
you can use this flag to turn the overlay section off, which would allow you to directly blit the original framebuffer to the screen if you wished, but the status bar is kind of important